sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py

```diff
@@ -231,8 +246,7 @@ def list_tables(
 
     @distributed_trace
     def query_tables(
-            self,
-            filter,  # pylint: disable=W0622
+            self, filter,  # type: str    pylint: disable=W0622
```

Comment:
nit: since this is async, we can use the python 3+ style type hints. I think I saw we have an issue open for that already.